<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SessionManagerWidget</class>
 <widget class="QWidget" name="SessionManagerWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>628</width>
    <height>253</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>IM</string>
  </property>
  <layout class="QVBoxLayout" name="mainVerticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <widget class="QFrame" name="MainFrame">
     <property name="font">
      <font>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QFrame" name="inputFrame">
        <property name="lineWidth">
         <number>0</number>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <property name="margin">
          <number>0</number>
         </property>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_input">
           <item>
            <widget class="QLabel" name="inputQuestionLabel">
             <property name="text">
              <string>Question label</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="inputQuestionLineEdit"/>
           </item>
           <item>
            <widget class="QPushButton" name="inputQuestionButtonOk">
             <property name="text">
              <string>OK</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="inputQuestionButtonCancel">
             <property name="text">
              <string>Cancel</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="usernameLabel">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>24</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Estrangelo Edessa</family>
            <pointsize>14</pointsize>
           </font>
          </property>
          <property name="text">
           <string>username@server.com</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="statusMessageLabel">
          <property name="font">
           <font>
            <family>Verdana</family>
            <pointsize>8</pointsize>
           </font>
          </property>
          <property name="text">
           <string>status message</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="statusIconLabel">
          <property name="minimumSize">
           <size>
            <width>20</width>
            <height>20</height>
           </size>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QTabWidget" name="sessionsTabWidget">
        <property name="styleSheet">
         <string notr="true"> QTabWidget::pane { /* The tab widget frame */
     border: 1px solid #9B9B9B;
 }

 QTabWidget::tab-bar {
     /*left: 5px;  move to the right by 5px */
 }

 /* Style the tab using the tab sub-control. Note that
     it reads QTabBar _not_ QTabWidget */
 QTabBar::tab {
     background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                 stop: 0 #E1E1E1, stop: 0.4 #DDDDDD,
                                 stop: 0.5 #D8D8D8, stop: 1.0 #D3D3D3);
     border: 1px solid #9B9B9B;
	 border-bottom: 0px;
     border-top-left-radius: 4px;
     border-top-right-radius: 4px;
     min-width: 8ex;
     padding: 2px;
     padding-right: 5px;
     padding-left: 1px;
 }

 QTabBar::tab:selected, QTabBar::tab:hover {
     background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                 stop: 0 #fafafa, stop: 0.4 #f4f4f4,
                                 stop: 0.5 #e7e7e7, stop: 1.0 #fafafa);
 }

 QTabBar::tab:selected {
     border-color: #9B9B9B;
     border-bottom-color: rgba(0,0,0,0); /* same as pane color */
 }

 QTabBar::tab:!selected {
     margin-top: 3px; /* make non-selected tabs look smaller */
 }

 /* make use of negative margins for overlapping tabs */
 QTabBar::tab:selected {
     /* expand/overlap to the left and right by 4px */
     margin-left: -2px;
     margin-right: -2px;
 }

 QTabBar::tab:first:selected {
     margin-left: 0; /* the first selected tab has nothing to overlap with on the left */
 }

 QTabBar::tab:last:selected {
     margin-right: 0; /* the last selected tab has nothing to overlap with on the right */
 }

 QTabBar::tab:only-one {
     margin: 0; /* if there is only one tab, we don't want overlapping margins */
 }</string>
        </property>
        <property name="currentIndex">
         <number>0</number>
        </property>
        <property name="movable">
         <bool>true</bool>
        </property>
        <widget class="QWidget" name="DefaultTab">
         <property name="styleSheet">
          <string>QWidget {
	background-color: #f4f4f4;
}

QLabel {
	color: rgb(0, 0, 0);
}</string>
         </property>
         <attribute name="title">
          <string>  Connected</string>
         </attribute>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="margin">
           <number>15</number>
          </property>
          <item>
           <widget class="QLabel" name="infoText_1">
            <property name="font">
             <font>
              <family>Estrangelo Edessa</family>
              <pointsize>12</pointsize>
             </font>
            </property>
            <property name="styleSheet">
             <string>color: rgb(69, 159, 255);</string>
            </property>
            <property name="text">
             <string>Hello and Welcome to Naali Communications Widget!</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="infoText_2">
            <property name="text">
             <string>You can find all important functionality from the menu that is located at the top of this window.</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="infoText_3">
            <property name="text">
             <string>To start click the Show Friend List button. There you can start chat,  video or audio sessions with your friends.</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
